pkg/snowflake/table.go

```diff
@@ -0,0 +1,145 @@
+package snowflake
+
+import (
+	"fmt"
+	"strings"
+)
+
+// TableBuilder abstracts the creation of SQL queries for a Snowflake Table
+type TableBuilder struct {
+	name      	string
+	db        	string
+	schema    	string
+	columns		map[string]string // {column_name: column_type}
```

Comment:
Maps in Go are deliberately randomised, so you'll have different order when you loop over them every time you run the loop. It looks like for columns in a table this is not a good fit, so consider having a separate type and have slice of those.

```
type column struct {
	name     string
	datatype string
	comment  string
}


...
columns []column
...

```